core/src/autogluon/core/models/abstract/abstract_model.py

```diff
@@ -299,12 +300,17 @@ def _preprocess_fit_resources(self, silent=False, **kwargs):
         return kwargs
 
     def fit(self, **kwargs):
-        kwargs = self._preprocess_fit_args(**kwargs)
-        if 'time_limit' not in kwargs or kwargs['time_limit'] is None or kwargs['time_limit'] > 0:
-            self._fit(**kwargs)
-        else:
-            logger.warning(f'\tWarning: Model has no time left to train, skipping model... (Time Left = {round(kwargs["time_limit"], 1)}s)')
-            raise TimeLimitExceeded
+        with warnings.catch_warnings():
+            # Suppress numpy 1.20 warnings (downstream scipy is not updated yet)
+            # https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
+            warnings.filterwarnings('ignore', message='`np.*` is a deprecated alias for the builtin `.*`')
```

Comment:
Only these are new, the rest of the changes are just indentation shift to the right